docs/source/el7toel8/actor-rhel7-to-rhel8.md

```diff
@@ -328,6 +328,17 @@ data is stored in the database for the current session:
 snactor run --save-output IPUWorkflowConfig
 ```
 
+Because of a bug in leapp, this does not work and gives the following error:
```

Comment:
Technically it's not a bug or something can be fixed for good, as snactor is a development tool and is not being run on the machine-to-be-upgraded, so it has no knowledge of what system leapp is planned to be run for and thus needs this information to be passed this way ot another. Setting `LEAPP_UPGRADE_PATH_TARGET_RELEASE` and `LEAPP_UPGRADE_PATH_FLAVOR` to the values you mentioned is the way to say "Run leapp actors as you would for a 7to8 upgrade".

Could you please rephrase the description a bit? Maybe something like `After leapp started supporting several upgrade paths snactor requires target system and upgrade flavor specification for a correct execution. To get around the possible leapp ModelViolationError ... please set the following environment variables prior to running snactor`.

And thanks a lot for the patch, that's definitely an awesome addition to the dev docs!

Reply:
Thanks for the feedback. I reworded the changed based on your suggestion.

Giving this another though, would it make sense to have this information somewhere inside of the `snactor`, when it is executed without these variables, reporting that it might not work for all the workflow related tasks, at least as a warning? Or even make it mandatory for execution of `snactor`? Or provide some user-friendly commandline switches for `snactor` to set this for you?

Reply:
You are definitely right, we should add this information to snactor.